app/views/coronavirus_landing_page/components/hub/_page_header.html.erb

```diff
@@ -24,7 +24,7 @@
           title: title[:text],
           inverse: true,
           margin_top: 4,
-          margin_bottom: 0,
+          margin_bottom: 6,
```

Comment:
Why did this margin change?

Reply:
Good spot - looks like I added this because I removed some css unintentionally, I have updated now